cnames_active.js

```diff
@@ -2218,6 +2218,7 @@ var cnames_active = {
   "nic": "nic.github.io",
   "nice": "gcsboss.github.io/nice",
   "nicetix": "indus.github.io/nicetix",
+  "niel": "niel-n7els-projects.vercel.app",
```

Comment:
```suggestion
  "niel": "cname.vercel-dns.com", // noCF
```
When using Vercel, you'll want to target `cname.vercel-dns.com` with the Cloudflare proxy disabled